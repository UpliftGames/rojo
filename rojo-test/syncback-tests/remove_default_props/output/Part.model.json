{
  "className": "Part",
  "properties": {
    "Size": [
      1.0,
      2.0,
      3.0
    ],
    "Anchored": true,
    "Massless": true
  }
}
