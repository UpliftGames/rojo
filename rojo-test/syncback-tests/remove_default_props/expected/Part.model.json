{
  "className": "Part",
  "properties": {
    "Size": [
      1.0,
      2.0,
      3.0
    ]
  }
}
